dolphinscheduler-alert/dolphinscheduler-alert-plugins/dolphinscheduler-alert-pagerduty/src/main/java/org/apache/dolphinscheduler/plugin/alert/pagerduty/PagerDutySender.java

```diff
@@ -53,21 +54,28 @@ public AlertResult sendPagerDutyAlter(String title, String content) {
         alertResult.setStatus("false");
         alertResult.setMessage("send pager duty alert fail.");
 
-        try {
-            sendPagerDutyAlterV2(alertResult, title, content);
-        } catch (Exception e) {
-            log.info("send pager duty alert exception : {}", e.getMessage());
+        for (int i = 0; i < AlertConstants.RETRY_TIMES; i++) {
+            try {
+                int statusCode = sendPagerDutyAlterV2(alertResult, title, content);
+                if (statusCode != AlertConstants.DEFAULT_VALUE) {
```

Comment:
hi @qingwli  when ```statusCode != HttpStatus.SC_OK``` && ```statusCode != AlertConstants.DEFAULT_VALUE```,do we need retry?

Reply:
No, if got statusCode not HttpStatus.SC_OK but actually got a code means the request is correct but got the wrong response, it's not network issue, it's config error